src/flamenco/runtime/fd_executor.c

```diff
@@ -1088,14 +1088,25 @@ fd_executor_setup_borrowed_accounts_for_txn( fd_exec_txn_ctx_t * txn_ctx ) {
   ulong j = 0;
   for( ulong i = 0; i < txn_ctx->accounts_cnt; i++ ) {
     fd_pubkey_t * acc = &txn_ctx->accounts[i];
+    txn_ctx->nonce_accounts[i] = 0;
 
     fd_borrowed_account_t * borrowed_account = fd_borrowed_account_init( &txn_ctx->borrowed_accounts[i] );
-    fd_acc_mgr_view( txn_ctx->acc_mgr, txn_ctx->funk_txn, acc, borrowed_account );
+    int account_nonexistent = fd_acc_mgr_view( txn_ctx->acc_mgr, txn_ctx->funk_txn, acc, borrowed_account );
```

Comment:
can you not handle the error this way. 

Prefer something like int err = fd_acc_mgr_view(..)
int is_unknown_account = err == FD_UNKNOWN_...
if( FD_UNLIKELY( other_err ) ) { FD_LOG_ERR(()); }